### PR TITLE
Only set one-off contribution cookie for one-off contributions

### PR DESCRIPTION
**Pull request by jranks123**
## Why are you doing this?

Previously, we were setting the one-off contribution cookie (that `gu.com` uses to prevent displaying the epic or banner) for recurring contributors hitting the thank you page. There is an equivalent recurring contributions cookie, which is set server side, so we don't need to set anything here for recurring contributors. 

The impact of this bug is that everyone who has given a recurring contribution in the past few weeks will have both cookies. There's nothing we can really do about this, other than fix it going forwards

### CODE REVIEW

File: assets/pages/new-contributions-landing/contributionsLanding.jsx [hunk 1]
```diff
@@ -81,10 +89,10 @@ const router = (
           exact
           path="/:countryId(uk|us|au|eu|int|nz|ca)/thankyou"
           render={() => {
-            setCookie(
-              ONE_OFF_CONTRIBUTION_COOKIE,
-              currentTimeInEpochMilliseconds.toString(),
-            );
+            // we set the recurring cookie server side
```
Comment (johnduffell) [inline assets/pages/new-contributions-landing/contributionsLanding.jsx]: good comment

File: assets/pages/new-contributions-landing/contributionsLandingInit.js [hunk 2]
```diff
@@ -84,6 +85,9 @@ const init = (store: Store<State, Action, Function>) => {
   selectDefaultPaymentMethod(state, dispatch);
   initialisePaymentMethods(state, dispatch);
   initialiseSelectedAnnualAmount(state, dispatch);
+  storage.setSession('contributionType', state.page.form.contributionType);
+  storage.setSession('paymentMethod', state.page.form.paymentMethod);
```
Comment (johnduffell) [inline assets/pages/new-contributions-landing/contributionsLandingInit.js]: what are these for?

**Comment by johnduffell** [comment]
good spot, thanks for fixing 👍

File: assets/pages/new-contributions-landing/contributionsLandingInit.js (hunk 2 above)
Comment (jranks123) [reply assets/pages/new-contributions-landing/contributionsLandingInit.js]: It's used in various places. We already set these when they change, this just sets them on initialisation too

**Pull request merged**